dev/Sidrah-Madiha/allcustommodules.py

```diff
@@ -0,0 +1,138 @@
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn import metrics
+from math import sqrt
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sn
+from sklearn.metrics import confusion_matrix, classification_report
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.model_selection import cross_val_score, GridSearchCV
+from sklearn.naive_bayes import GaussianNB
+from sklearn.svm import SVC
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.linear_model import LogisticRegression
+from matplotlib.pyplot import figure
+import seaborn as sns
+from sklearn.svm import SVC
+
+    
+def data_stats(dataset):
+    ''' Shows some basic stats of the dataset'''
+    print("=========== SOME STATS of Dataset ===========")
+    print('Shape of the dataset: ' + str(dataset.shape) + "\n")
+    print('List of attribute columns' , list(dataset.columns))
+    print("\n")
+    list_cat = dataset.Class.unique()
+    print('List of Categories ' , list_cat , "\n" )
+    
+            
+def tokenize_target_column(dataset):
+    ''' tokenize the Class column values to numeric data'''
+    factor = pd.factorize(dataset['Class'])
+    dataset.Class = factor[0]
+    definitions = factor[1]
+    print("Updated tokenize 'Class' column - first 5 values")
+    print(dataset.Class.head())
+    print("Distinct Tokens used for converting Class column to integers")
+    print(definitions)
+    return definitions
+def train_data_test_data_split(dataset):
```

Comment:
please include a docstring explaining the functionality of `train_data_test_data_split`